pymongo/database.py

```diff
@@ -292,6 +292,147 @@ def get_collection(
             read_concern,
         )
 
+    def _get_encrypted_fields(self, kwargs, coll_name, ask_db):
```

Comment:
Are there any existing places where we could use this method to avoid duplication?